serbian/burekSir/pocetak/pocetak.md

```diff
@@ -0,0 +1,20 @@
+Čemu [bes](bes.md), čemu [ozlojađenost](nezadovoljstvo.md). Pekara može i sme da ostane bez mog omiljenog bureka. Ljudi oko mene mogu da se smeju. Loše stvari mogu da se dese. Dobre stvari mogu da se dese. 

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.